### PR TITLE
gh-123843: Remove broken links to the Zope DateTimeWiki

### PR DESCRIPTION
**Pull request by AA-Turner**
As an alternative to #123844.

cc: @otkd @Eclips4 

A

* Issue: gh-123843

### CODE REVIEW

**Comment by Eclips4** [comment]
If we're talking about "Are these links really useful anymore?" I would say "No, let's remove them".

So I'm in favor of removing them.

File: Lib/datetime.py [hunk 1]
```diff
@@ -1,9 +1,13 @@
+"""Concrete date/time and related types.
```
Comment (otkd) [inline Lib/datetime.py]:
```suggestion
"""Specific date/time and related types.
```

File: Lib/datetime.py [hunk 2]
```diff
@@ -1,9 +1,13 @@
...
+
+See http://www.iana.org/time-zones/repository/tz-link.html for
```
Comment (otkd) [inline Lib/datetime.py]:
```suggestion
See https://data.iana.org/time-zones/tz-link.html for
```
Replaced with resolved link

**Review by otkd — CHANGES_REQUESTED** [review]
Minor change for clarity and resolved link

**Review by otkd — APPROVED** [review]
Thanks!

**Comment by terryjreedy** [comment]
Looks good; nothing to add.

**Pull request merged**